app/Main.hs

```diff
@@ -81,22 +78,22 @@ main =
               else do
                 upd                      <- newMVar () -- putMVar when to update
                 mgr                      <- newTlsManager
-                logHandle <- openFile (view cLog config) AppendMode
                 (viewRef, refreshAction) <- runLoggingT
                   (runReaderT refresh (ClientContext config mgr))
                   print
-                Async.concurrently_ 
-                  (Async.concurrently_ 
+
+                Async.concurrently_
```

Comment:
I added a newline and suddenly the differ goes all excited.

Reply:
It had spaces after the underscore before, so that's why it is getting so excited

Reply:
That explains it! Thank you.